framework/test/jest/integration/specs/modules/chain/rebuilding.spec.js

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright © 2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+'use strict';
+
+describe('rebuilding', () => {
+	describe('rebuilding to end of round 2 when blockchain contains 303 blocks', () => {
```

Comment:
Maybe remove extra describe or add more scenarios if the idea was that there might be more?